index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <p class="title">Hello world</p> 
+</body>
+</html>
```

Comment:
add an empty line at the end of file, your text editor has a config todo this automatically, please turn it on to avoid this things in the future.